drivers/i2c/i2c_ll_stm32.meta.jinja2

```diff
@@ -0,0 +1,36 @@
+{# Copyright (c) 2018 Nordic Semiconductor ASA #}
+{# SPDX-License-Identifier: Apache-2.0         #}
+
+{% extends 'device_declare.jinja2' %}
+
+{% block init_params %}
```

Comment:
Is there some other way to do this?  Can we use a macro that we can pass these values to?

Reply:
Seems like I still need the block to ensure the order of when the template is rendered, but I now have version with a set/get pair.